### PR TITLE
🌐 [i18n-KO] Translated `model_doc/bartpho.md` to Korean

### PR DESCRIPTION
**Pull request by Jwaminju**
# What does this PR do?

Translated the `model_doc/bartpho.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@junejae, @Jwaminju, @010kim, @4N3MONE, @boyunJang 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)
@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/model_doc/bartpho.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+>>> bartpho = AutoModel.from_pretrained("vinai/bartpho-syllable")
+
+>>> tokenizer = AutoTokenizer.from_pretrained("vinai/bartpho-syllable")
+
+>>> line = "Chúng tôi là những nghiên cứu viên."
+
+>>> input_ids = tokenizer(line, return_tensors="pt")
+
+>>> with torch.no_grad():
+...     features = bartpho(**input_ids)  # Models outputs are now tuples
```
Comment (4N3MONE) [inline docs/source/ko/model_doc/bartpho.md:49]: 주석 부분도 번역이 필요할 것 같습니다! (애매하네요 😅)

**Review by 4N3MONE — COMMENTED** [review]
번역 고생하셨습니다! 리뷰 1개 올렸습니다.

File: docs/source/ko/model_doc/bartpho.md (hunk 1 above)
Comment (Jwaminju) [inline docs/source/ko/model_doc/bartpho.md:49]:
```suggestion
...     features = bartpho(**input_ids)  # 이제 모델 출력은 튜플입니다
```

Comment (Jwaminju) [reply docs/source/ko/model_doc/bartpho.md]: 감사합니다! 수정했습니다!

**Review by stevhliu — APPROVED** [review]
LGTM, thanks! 🤗

File: docs/source/ko/_toctree.yml [hunk 2]
```diff
@@ -320,8 +320,8 @@
         title: (번역중) BART
       - local: in_translation
         title: (번역중) BARThez
-      - local: in_translation
-        title: (번역중) BARTpho
+      - local: model_doc/bartpho.md
```
Comment (stevhliu) [inline docs/source/ko/_toctree.yml:323]:
```suggestion
      - local: model_doc/bartpho
```

**Pull request merged**